src/backer/backering.py

```diff
@@ -221,28 +230,6 @@ def cueDo(self, tymth=None, tock=0.0):
                 yield self.tock
             yield self.tock
 
-    def exchangerDo(self, tymth=None, tock=0.0):
-        """
-        Returns doifiable Doist compatibile generator method (doer dog) to process
-            .exc responses and pass them on to the HTTPRespondant
-
-        Parameters:
-            tymth (function): injected function wrapper closure returned by .tymen() of
-                Tymist instance. Calling tymth() returns associated Tymist .tyme.
-            tock (float): injected initial tock value
-
-        Usage:
-            add result of doify on this method to doers list
-        """
-        self.wind(tymth)
-        self.tock = tock
-        _ = (yield self.tock)
-
-        while True:
-            for rep in self.exc.processResponseIter():
-                self.replies.append(rep)
-                yield  # throttle just do one cue at a time
-            yield
```

Comment:
Hmm, why is this removed, is it actually not needed for the backer to work?

Reply:
the processResponseIter outdated, and remove from the Exchanger
https://github.com/WebOfTrust/keripy/blob/3febc54a463125a4cc3d86d694e57004134b2092/src/keri/peer/exchanging.py#L22

Reply:
Also take a look at keripy, keria, it seems that exchangeDo, processResponseIter are no longer in used

Reply:
got it, checked the history and this was removed during a major refactor of exchange messages, all good